guidelines.md

```diff
@@ -2,6 +2,12 @@
 
 1. No spamming. Spamming is the repeated uploading of the same or similar content over and over for where there is no real effort put into the content.
 
-1. Do not post NSFW or suggestive minors, including drawings; do not participate in hate speech, violent extremism; do not share malware; do not promote self-harm or suicide; do not harass users or evade blocks/bans.
+2. Do not post NSFW or/of suggestive minors, including drawings; **"B-But fiction doesn't affect reality"** well it seems like its affecting the reality of your cock and balls.
```

Comment:
```suggestion
2. Do not post NSFW or/of suggestive minors, including drawings
```
I believe you did a little bit of trolling :trollface:

Reply:
> I believe you did a little bit of trolling :trollface:

:nerd_face: